src/main/java/com/tileman/TilemanModeOverlay.java

```diff
@@ -99,10 +99,12 @@ private void drawTile(Graphics2D graphics, WorldPoint point)
 	}
 
 	private Color getTileColor() {
-		if (plugin.getRemainingTiles() <= 0) {
-			return Color.RED;
-		} else if (plugin.getRemainingTiles() <= config.warningLimit()) {
-			return Color.ORANGE;
+		if(config.enableTileWarnings()) {
+			if (plugin.getRemainingTiles() <= 0 && config.enableTileWarnings()) {
```

Comment:
don't need the `&& config.enableTileWarnings()` here

Reply:
Removed